api/client/proto/authservice.proto

```diff
@@ -1610,6 +1662,16 @@ service AuthService {
     // password token.
     rpc CreateResetPasswordToken(CreateResetPasswordTokenRequest) returns (types.UserTokenV3);
 
+    // CreateBot creates a new bot user.
+    rpc CreateBot(CreateBotRequest) returns (CreateBotResponse);
+    // DeleteBot deletes a bot user.
+    rpc DeleteBot(DeleteBotRequest) returns (google.protobuf.Empty);
+    // GetBotUsers gets all users with bot labels.
+    rpc GetBotUsers(GetBotUsersRequest) returns (stream types.UserV2);
```

Comment:
Curious, why does this leverage streaming?

Reply:
`GetUsers()` does and I just cloned its declaration